js/preloader.js

```diff
@@ -0,0 +1,6 @@
+
+$(document).ready(function() {
```

Comment:
These lines of code can be written without jquery. (Did not test it but this should be it:)
```
window.addEventListener("load", function() {
    document.body.classList.add("loaded");
});
```
One CAN include some more megabytes of code but I like to avoid that, also looking at the loading time you like to optimize.

What do you think about this? As a reminder I am not telling you to do something to get this merged. I would like to have a conversation.

Reply:
Yes, you are right. I forgot it. It's in general not a good practice to implement `jQuery`.

Reply:
Note for me: Fixed!